DFC.App.DiscoverSkillsCareers.Services/Services/ResultsService.cs

```diff
@@ -74,14 +78,43 @@ public async Task<GetResultsResponse> GetResultsByCategory(string jobCategoryNam
                 .Select(x => x.First())
                 .ToList();
 
+            var allJobCategories =
+                await jobProfileCategoryDocumentService.GetAsync(document => document.PartitionKey == "JobProfileCategory")
```

Comment:
Should this be a case insensitive equality check?

Reply:
I think the way these are written it will never be cased differently - and it would mean changing a bunch for consistency so will leave it if you dont mind - thx